netlify/edge-functions/card.tsx

```diff
@@ -1,8 +1,5 @@
-// @ts-ignore
```

Comment:
Not sure why this didn't work for you the other day @foxyblocks, but having these VS Code settings removes the need for the @ts-ignores below.